README.md

```diff
@@ -22,6 +22,12 @@ This program serves two purposes:
 2. Once we figure out all the details, it should be useful as an
    example of a systemd generator in rust.
 
+### Installation
+
+The "{generator}" template `swap-create@.service` must be substituted
+for the final location of the zram-generator binary and copied into
+a system unit directory.
```

Comment:
Doesn't cargo have a way to install data files? Telling users to put files in various locations is very error prone.

Reply:
It doesn't, owing to the fact that cargo is just a libdep resolver (and you can build an executable if you absolutely *must*), and not a distribution/package manager.

I usually just use makefiles.

Reply:
OK. So let's just hardcode the value in the template.